docker/minfi/1.48.0-1/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/biocontainers/bioconductor-minfi:1.48.0--r43hdfd78af_0
+
+RUN R --no-save <<SCRIPT
+    if (!require("BiocManager", quietly = TRUE))
+        install.packages("BiocManager", repos='http://cran.us.r-project.org')
+    BiocManager::install("IlluminaHumanMethylationEPICmanifest")
+    BiocManager::install("IlluminaHumanMethylationEPICanno.ilm10b4.hg19")
```

Comment:
Those two R packages wrap the Illumina manifest files for the EPIC array (https://support.illumina.com/downloads/infinium-methylationepic-v1-0-product-files.html) and the second adds the genomics coordinates (in hg19, because that's what Illumina provides). The source CSV is ~500MB and contains a lot of metadata for each probe that is found on the EPIC array. They're necessary for the steps in `preprocess`, particularly `MapToGenome`.

Reply:
So this is essentially downloading reference files? Should these be included in the docker image?

Reply:
Sort of, but they're R packages that the underlying `minfi` functions expect to exist. So there isn't a way to provide just the Illumina manifest as a file.